packages/opentelemetry-exporter-zipkin/src/zipkin.ts

```diff
@@ -109,7 +106,11 @@ export class ZipkinExporter implements SpanExporter {
     const zipkinSpans = spans.map(span =>
       toZipkinSpan(
         span,
-        serviceName,
+        String(
```

Comment:
why `String` ?
`serviceName` param will always have the same value, no reason to have it as 2nd param, you can use here `this.DEFAULT_SERVICE_NAME` always.

Reply:
It doesn't type-check without `String`, because the values in `attributes` can be numbers.

I reverted the change to computing `this._serviceName` to preserve backwards compatibility for persisting the service name from the first span's resource, so I'll leave the param here.